src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -64,25 +64,38 @@ public BlobManager(
             if (cloudBlockBlob.getProperties().getLeaseStatus() == LeaseStatus.LOCKED) {
                 log.info(LEASE_ALREADY_ACQUIRED_MESSAGE, zipFilename, containerName);
                 return Optional.empty();
-            } else if (!readyToAcquireLease(cloudBlockBlob)) {
+            } else if (!readyToAcquireLease(cloudBlockBlob, zipFilename, containerName)) {
                 log.info(
-                    "Can't acquire lease on file {} in container {} "
-                        + "because lease was acquired less than {} seconds ago.",
+                    "Lease already acquired on file {} in container {} less than {} seconds ago. "
+                        + "Lease Expires at: {}",
                     zipFilename,
                     containerName,
-                    properties.getBlobLeaseAcquireDelayInSeconds()
+                    properties.getBlobLeaseAcquireDelayInSeconds(),
+                    cloudBlockBlob.getMetadata().get(LEASE_EXPIRATION_TIME)
                 );
                 return Optional.empty();
             }
 
             String leaseId = cloudBlockBlob.acquireLease(properties.getBlobLeaseTimeout(), null);
-            log.info("Acquired lease on file {} in container {}. Lease ID: {}", zipFilename, containerName, leaseId);
```

Comment:
lets keep this one until we sure upload is working well

Reply:
This log is not removed. just added expiration time. see the line number 92

Reply:
instead of logging it after uploading logging it just after lock might be better

Reply:
I've updated to keep the existing log message as it was before. Added a new log after updating the metadata